packages/inferno-vnode-flags/src/index.ts

```diff
@@ -17,8 +18,14 @@ const enum VNodeFlags {
 	Void = 1 << 12,
 
 	FormElement = InputElement | TextareaElement | SelectElement,
-	Element = HtmlElement | SvgElement | MediaElement | InputElement | TextareaElement | SelectElement,
+	Element = HtmlElement |
```

Comment:
UnionTypes look like this btw:

```js
type Element =
  | HtmlElement
  | SvgElement
  | MediaElement
  | InputElement
  | TextareaElement
  | SelectElement
```

which is not valid for binary or, as you can't have leading operators for them.